web/src/features/charts/bar-breakdown/GraphCard.tsx

```diff
@@ -1,6 +1,6 @@
 import { twMerge } from 'tailwind-merge';
 
-export function GraphCard({
+export function RoundedCard({
```

Comment:
Be nice to keep the file name the same as the exported component when possible 🛠️